Assets/Scrips/CameraMovement.cs

```diff
@@ -3,20 +3,24 @@
 
 public class CameraMovement : MonoBehaviour {
 
-	public const float smoothness = 0.5f;         
-	private Vector2 playerVelocity;
+	public const float smoothness = 0.7f;         
 	public GameObject target; 
```

Comment:
To rid yourself of "GetComponent()" calls, change this variable type to "RigidBody2d".